src/generic/editlbox.cpp

```diff
@@ -121,38 +121,42 @@ bool wxEditableListBox::Create(wxWindow *parent, wxWindowID id,
     wxPanel *subp = new wxPanel(this, wxID_ANY, wxDefaultPosition, wxDefaultSize,
                                 wxSUNKEN_BORDER | wxTAB_TRAVERSAL);
     wxSizer *subsizer = new wxBoxSizer(wxHORIZONTAL);
-    subsizer->Add(new wxStaticText(subp, wxID_ANY, label), 1, wxALIGN_CENTRE_VERTICAL | wxLEFT, 4);
+
+    subsizer->Add(new wxStaticText(subp, wxID_ANY, label),
+                  wxSizerFlags(1).Center().Border(wxLEFT));
+
+    const wxSizerFlags centered = wxSizerFlags().Center();
 
     if ( m_style & wxEL_ALLOW_EDIT )
     {
         m_bEdit = new wxBitmapButton(subp, wxID_ELB_EDIT,
                                      wxArtProvider::GetBitmap(wxART_EDIT, wxART_BUTTON));
-        subsizer->Add(m_bEdit, 0, wxALIGN_CENTRE_VERTICAL);
+        subsizer->Add(m_bEdit, centered);
     }
 
     if ( m_style & wxEL_ALLOW_NEW )
     {
         m_bNew = new wxBitmapButton(subp, wxID_ELB_NEW,
                                     wxArtProvider::GetBitmap(wxART_NEW, wxART_BUTTON));
-        subsizer->Add(m_bNew, 0, wxALIGN_CENTRE_VERTICAL);
+        subsizer->Add(m_bNew, centered);
     }
 
     if ( m_style & wxEL_ALLOW_DELETE )
     {
         m_bDel = new wxBitmapButton(subp, wxID_ELB_DELETE,
                                     wxArtProvider::GetBitmap(wxART_DELETE, wxART_BUTTON));
-        subsizer->Add(m_bDel, 0, wxALIGN_CENTRE_VERTICAL);
+        subsizer->Add(m_bDel, centered);
```

Comment:
Is this really more readable than repeating `wxSizerFlags().Center()` in all uses? I'd probably pick the latter myself for unsurprising readability.

Reply:
I think it would make it more clear, yes.